pages/index.jsx

```diff
@@ -3,10 +3,34 @@ import NFTGallery from "../components/nftGallery";
 
 export default function Home() {
   return (
-    <div>
+    <div className="bg-purple">
       <main className={styles.main}>
         <NFTGallery />
       </main>
+      
+
+      <div className="stars">
```

Comment:
This complete div should be in separate file to maintain the consistency  of the code. @Navansh